packages/apputils/src/sessioncontext.tsx

```diff
@@ -266,11 +277,12 @@ export namespace ISessionContext {
     readonly autoStartDefault?: boolean;
   }
 
-  export type KernelDisplayStatus =
-    | Kernel.Status
-    | Kernel.ConnectionStatus
-    | 'initializing'
-    | '';
+  export type KernelDisplayStatus = string;
+  // export type KernelDisplayStatus =
```

Comment:
Should this be removed?

Reply:
Ah yes I forgot... so the Kernel status was hardcoding the status actual text in the type... .so we cannot localize it. So I just made kernel display status be a string... any other way to handle this?

Reply:
Fixed.

Reply:
It comes from the server but there are a finite number of them, so we could translate them all.

Reply:
Or should I turn this type into an object with a status (that can remain the same as it is) and a displayStatus that is the translated string?

Reply:
I like displayStatus

Reply:
So to double check. Initialize this object like?

`{[status: KernelDisplayStatus]: string}`  somewhere and the use it to get the localized display status where needed?

Reply:
Hmm, I think `getKernelDisplayStatus()` returns a translated `string` now that you mention it.